### ISSUE TITLE
Gyroscope iphone 13 pro

### ISSUE THREAD

**Issue by sevenor** [mistic100/Photo-Sphere-Viewer#765]
We check the gyroscope on the iphone 13 pro and the picture is constantly twitching when moving. As if it's a stabilization glitch. How can you make the movements straight and clear?

**Comment by mistic100** [comment]
I have no way to test on an iphone. Such problems will have to be solved by the community. Sorry.

**Comment by sevenor** [comment]
We check the gyroscope on the android and all also too. Glitch

**Comment by mistic100** [comment]
I don't look at issues without details as stated in the [issue guidelines](https://github.com/mistic100/Photo-Sphere-Viewer/blob/dev/.github/ISSUE_TEMPLATE.md) you should have read when posting.

**Comment by sevenor** [comment]
Check pls this page https://www.ilyaeliseev.ru/dizain-interyera/potapovo-life#image=1451

**Comment by sevenor** [comment]
If you follow this link, you can notice twitching from any phone. Maybe the problem is in the settings?

**Comment by mistic100** [comment]
Yes i see issues but absolutely not with the viewer. Your integration with a slider is broken and does not allow to navigate in the sphere, insteads it slides to the next picture. Plus there is no "Gyroscope" button.

I will not try to analyze your code to tell what is wrong. I will only do so (in a new issue) if you reproduce the issue with a single autonomous viewer without all the clutter of your website.

**Comment by sevenor** [comment]
Flipping through the slider is absolutely not necessary.
If you log in from your phone, the gyro button will be. And if you press it, it will turn on and moving the phone, you can see everything

If anything, my 360 call code is like this:
`if(dataurl){
			const viewer = new PhotoSphereViewer.Viewer({
				container: dataurl.querySelector('.viewer'),
				panorama: dataurl.getAttribute("data-url"),
				navbar: [
				    'autorotate',
                    'gyroscope',
				].filter(v => v),
				lang: {
				  autorotate: 'Автоматическое вращение',
				  zoom: 'Увеличение',
				  zoomOut: 'Уменьшить',
				  zoomIn: 'Увеличить',
				  download: 'Скачать',
				  fullscreen: 'Полноэкранный режим',
				  menu: 'Меню',
				  twoFingers: 'Используйте два пальца для навигации',
				  ctrlZoom: 'Используйте ctrl + scroll для увеличения изображения',
				  loadError: 'Панорама не загружается',
				},
				loadingTxt: 'Загрузка...',
				autorotateDelay: 1,
				touchmoveTwoFingers: true,
				autorotateSpeed: '1rpm',
				defaultLong: '225',
				defaultZoomLvl: 50,
                plugins: [
                    [PhotoSphereViewer.GyroscopePlugin, {
                        touchmove       : true,
                        absolutePosition: true,
                        moveMode        : "fast",
                      }],
                  ],
			});
			if(document.body.clientWidth > 1023)
            {
                splider.options = {
                    drag: false
                };
            }
		}`

**Comment by mistic100** [comment]
still, there is no "twitching".
if you want more help create a isolated demo, a video, seomthing to explain what you mean
